mediapipe/tasks/python/vision/face_stylizer.py

```diff
@@ -54,11 +55,15 @@ class FaceStylizerOptions:
   """
 
   base_options: _BaseOptions
+  running_mode: _RunningMode = _RunningMode.IMAGE
```

Comment:
We don't support RunningMode in FaceStylizer. It must always be .IMAGE

Reply:
Fixed it now.